inc/alert.class.php

```diff
@@ -467,6 +467,7 @@ static function displayAlerts($params = []) {
       }
       if (!$p['show_only_login_alerts']
          && $alerts = self::findAllToNotify(['show_only_login_alerts' => false,
+                                             'show_only_helpdesk_alerts' => true,
                                              'show_hidden_alerts'     => true])
```

Comment:
Maybe this should be better to take care of the context.
```suggestion
      $hidden_params = [
         'show_hidden_alerts'        => true,
         'show_only_helpdesk_alerts' => $p['show_only_helpdesk_alerts'],
         'entities_id'               => $p['entities_id']
      ];
      if (!$p['show_only_login_alerts']
          && !$p['show_hidden_alerts']
          && $alerts = self::findAllToNotify($hidden_params)) {
```

Reply:
done